src/main/java/bhs/devilbotz/subsystems/Gripper.java

```diff
@@ -18,35 +19,45 @@
  * @author joshuamanoj &amp; ParkerMeyers
  */
 public class Gripper extends SubsystemBase {
-  private final DoubleSolenoid gripperSolenoid;
-  private static final Compressor pcmCompressor =
-      new Compressor(Constants.GripperConstants.COMPRESSOR_CAN_ID, PneumaticsModuleType.CTREPCM);
+  private DoubleSolenoid gripperSolenoid = null;
+  private Compressor pcmCompressor = null;
 
   /** The constructor for the gripper subsystem. */
   public Gripper() {
-    pcmCompressor.enableDigital();
-    pcmCompressor.disable();
+    if (RobotConfig.isCompBot()) {
+      pcmCompressor =
+          new Compressor(
+              Constants.GripperConstants.COMPRESSOR_CAN_ID, PneumaticsModuleType.CTREPCM);
+      pcmCompressor.enableDigital();
+      pcmCompressor.disable();
 
-    gripperSolenoid =
-        new DoubleSolenoid(
-            PneumaticsModuleType.CTREPCM,
-            Constants.GripperConstants.GRIPPER_SOLENOID_FORWARD,
-            Constants.GripperConstants.GRIPPER_SOLENOID_REVERSE);
+      gripperSolenoid =
+          new DoubleSolenoid(
+              PneumaticsModuleType.CTREPCM,
```

Comment:
This is outdated, please merge main into your branch.

Reply:
Hi @ParkerMeyers, Maciej's branch is updated already.  Which change is outdated?

Reply:
In the ArmFix branch there was new gripper code. I thought I comitted it, but I guess I did not. Ill create a PR. This is because we physically switched from a CTRE PCM to a REV PCM.

Reply:
Done. Please merge & Review #74 before merging this commit. @BrownGenius